cobench.opam

```diff
@@ -12,6 +12,7 @@ depends: [
   "yojson"
   "cohttp-lwt-unix"
   "odoc" {with-doc}
+  "bechamel" {with-test}
```

Comment:
Can you remove the `{with-test}` here? (since bechamel is actually essential to the library, not just for testing it)